release/lib/skippers.ts

```diff
@@ -12,6 +12,18 @@ export const skipIfBrowserSame = ({ packages }: Context) => {
     return '';
 };
 
+export const skipIfVSCodeSame = ({ packages }: Context) => {
+    const pkg = packages && packages.get('vscode-webhint');
```

Comment:
This looks a lot like `skipIfBrowserSame`. What about a `skipIfSameVersion` and do some currying?

```ts
const skipIfSameVersion = (pkgName: string) => {
    return ({ pakages }: Context) => {
       // magic here
    };
};

const skipIfBrowserSame = skipIfSameVersion('extension-browser');
const skipIfVSCodeSame = skipIfSameVersion('vscode-webhint');